actions.js

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 Uber Technologies, Inc.
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
@heshan0131 should we use __Uber Technologies__ still? or UCF?

Reply:
yea use Uber